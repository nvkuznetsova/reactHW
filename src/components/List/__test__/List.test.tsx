import React from 'react';
import { shallow } from 'enzyme';

import { List } from '..';

describe('List', () => {
  it('should match snapshot', () => {
    const children = <li>Test</li>;
    const component = shallow(<List>{children}</List>);

    expect(component).toMatchSnapshot();
  });
});