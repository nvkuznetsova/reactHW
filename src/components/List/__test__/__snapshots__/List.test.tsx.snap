// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List should match snapshot 1`] = `
<ul
  className="list-group "
>
  <li>
    Test
  </li>
</ul>
`;
